src/vex.js

```diff
@@ -0,0 +1,287 @@
+// String to DOM function
+var domify = require('domify')
+// classList polyfill for old browsers
+require('classlist-polyfill')
+
+// Object.assign polyfill
```

Comment:
Yes, it would! I switched to an npm module, [es6-object-assign](https://www.npmjs.com/package/es6-object-assign), which uses the same MDN code that I'm using. Commit: dadae50